app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt

```diff
@@ -71,6 +70,13 @@ private fun privateModeAdapterItems() = listOf(AdapterItem.PrivateBrowsingDescri
 private fun onboardingAdapterItems(onboardingState: OnboardingState): List<AdapterItem> {
     val items: MutableList<AdapterItem> = mutableListOf(AdapterItem.OnboardingHeader)
 
+    items.addAll(
```

Comment:
Why is this pull up & out of the `items.addAll()` below?

Reply:
It's because we want to change the order to match https://github.com/mozilla-mobile/fenix/issues/18494#issue-832328998


```
New order:

    Theme picker
    Address bar location picker
    Enhanced Tracking Protection settings
    Sync Firefox between devices
    Privacy notice
    Start browsing call to action

The diff is a bit confusing, if you see the whole file is will be a bit clear https://github.com/mozilla-mobile/fenix/blob/0cdb8a088aef815bb4f529b083335e2354285a80/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt

```